src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_launcher_integration_test.py

```diff
@@ -662,3 +664,119 @@ def test_merge_reductions(self, mock_get_timeout):
     """Tests merging. Wrapper around _test_merge_reductions."""
     mock_get_timeout.return_value = get_fuzz_timeout(1.0)
     self._test_merge_reductions('minijail-merge')
+
+
+@test_utils.integration
+@test_utils.with_cloud_emulators('datastore')
+class TestLauncherZFuchsia(BaseLauncherTest):
```

Comment:
This typo-looking nomenclature is actually a poor hack that I'd like guidance on fixing.

It turns out, due to (I believe) the way we're using environment variables, order implicitly matters for these integration tests--if Fuchsia tests run before Minijail tests, the Minijail tests will fail.

I *believe* this is because of OS_OVERRIDE and QUEUE_OVERRIDE getting set (a test that contains only those lines reproduces the bug; removing the lines at least allows Minijail to succeed).

The specific *manner* in which the failure manifests is that get_llvm_symbolizer_path() returns NoneType around libfuzzer.py:L553ish when you hit the Minijail tests.

I tried everything I could think of to "unset" these two env vars (del os.environ['variable'], os.environ.pop, environment.set_value('variable', '')), but nothing seemed to work (one example is still included in tearDown for this class)?

If anyone has any ideas for what might be happening here, let me know; I'd like to fix this a proper non-brittle way!

Reply:
Add `helpers.patch_environ()` (import `helpers` from `tests.test_libs`) to `setUp()`.

Reply:
perfect! thank you!